docs/pages/deployment/tls-configuration.rst

```diff
@@ -211,6 +211,14 @@ For `HAProxy <https://www.haproxy.com/>`_ the proxy configuration could look as
         http-request set-header X-SSL-CERT %{+Q}[ssl_c_der,base64]
         server node1 nuts_node:5555 check proto h2
 
+Revoked Certificates
+^^^^^^^^^^^^^^^^^^^^
+
+Proxies should always check whether the presented client certificate is revoked, e.g. in case its private was leaked.
+Many proxies don't automatically check certification revocation status unless explicitly configured.
+For HAProxy and NGINX you need to download/update the CRLs yourself and configure the proxy to use it (generally achieved using a scheduled script).
```

Comment:
Does the node update the CRL?

Reply:
Yes 🚀 
Fancy bloom-table backed CRL checker